coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,28 @@
+import logging
 import os
 
-from pyprint.ConsolePrinter import ConsolePrinter
-
 from coalib.output.Logging import configure_logging
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.parsing import Globbing
 from coalib.parsing.Globbing import glob_escape
 from coalib.settings.ConfigurationGathering import get_config_directory
 from coalib.settings.Section import Section
 
 
 def main(log_printer=None, section: Section=None):
+    """
+    >>> import logging
+    >>> from coalib.output.Logging import configure_logging
+    >>> configure_logging(stdout=True, color=False)
+    20
+    >>> main(log_printer=SomeLogPrinter)
+    [WARNING][... Using log_printer is deprecated...
```

Comment:
maybe we should move that into a normal test file, not so good as documentation ;)